pkg/ccl/sqlproxyccl/acl/file_test.go

```diff
@@ -38,6 +39,9 @@ func TestDenyListFileParsing(t *testing.T) {
 			{ClusterType, "cluster"},
 		}
 		for _, tc := range cases {
+			defer leaktest.AfterTest(t)()
+			testutilsccl.ServerlessOnly(t)
```

Comment:
Should this be outside of the for loop, given that `t` doesn't change across iterations?

Reply:
Fixed